src/config/ethereum/common.ts

```diff
@@ -0,0 +1,43 @@
+import { AppConfig } from 'config/types';
+
+export const commonConfig: AppConfig = {
+  mode: 'development',
+  appUrl: 'https://app.carbondefi.xyz',
+  carbonApi: 'https://api.carbondefi.xyz/v1/',
+  rpcUrl:
+    import.meta.env.VITE_CHAIN_RPC_URL ||
+    'https://eth-mainnet.alchemyapi.io/v2/41hlTcJUXX7kLw5oPejnQJ7V9BdW8Ysg',
```

Comment:
IMHO keys should not be deployed to github. Specially not our dev key for localhost usage. We had abuse in the past of our keys.